addon/templates/components/html-select.hbs

```diff
@@ -1,4 +1,4 @@
-<select {{action 'change' on='change'}} class="form-control">
+<select {{action 'change' on='change'}} class="{{mainComponent.elementClass}}">
```

Comment:
Btw. you know that your demo is broken? If you click on a menu element the menu disappears...